### PR TITLE
add support for custom buckets

### PR DESCRIPTION
**Pull request by alberhander**
- Support for S3 and GCS custom buckets
- Secrets refactor

### CODE REVIEW

File: chart/templates/import-api/secret.yaml [hunk 1]
```diff
@@ -11,10 +11,14 @@ metadata:
...
-stringData:
-  .secret.env: |
-    # Common configuration
-    # CARTO self hosted sensitive information
-    ENCRYPTION_SECRET_KEY={{ .Values.cartoSecrets.encryptionSecretKey }}
-    REACT_APP_GOOGLE_MAPS_API_KEY={{ .Values.cartoSecrets.googleMapsApiKey }}
+data:
+  # Common configuration
+  # CARTO self hosted sensitive information
+  ENCRYPTION_SECRET_KEY: {{ .Values.cartoSecrets.encryptionSecretKey }}
```
Comment (alvarorm22) [inline chart/templates/import-api/secret.yaml:17]: add `| quote` to avoid errors

File: chart/templates/import-worker/secret.yaml [hunk 2]
```diff
@@ -11,10 +11,14 @@ metadata:
...
-stringData:
-  .secret.env: |
-    # Common configuration
-    # CARTO self hosted sensitive information
-    ENCRYPTION_SECRET_KEY={{ .Values.cartoSecrets.encryptionSecretKey }}
-    REACT_APP_GOOGLE_MAPS_API_KEY={{ .Values.cartoSecrets.googleMapsApiKey }}
+data:
+  # Common configuration
+  # CARTO self hosted sensitive information
+  ENCRYPTION_SECRET_KEY: {{ .Values.cartoSecrets.encryptionSecretKey }}
```
Comment (alvarorm22) [inline chart/templates/import-worker/secret.yaml:17]: `| quote`

File: chart/templates/workspace-api/secret.yaml [hunk 3]
```diff
@@ -11,10 +11,16 @@ metadata:
...
-stringData:
-  .secret.env: |
-    # Common configuration
-    # CARTO self hosted sensitive information
-    ENCRYPTION_SECRET_KEY={{ .Values.cartoSecrets.encryptionSecretKey }}
-    REACT_APP_GOOGLE_MAPS_API_KEY={{ .Values.cartoSecrets.googleMapsApiKey }}
+data:
+  # Common configuration
+  # CARTO self hosted sensitive information
+  ENCRYPTION_SECRET_KEY: {{ .Values.cartoSecrets.encryptionSecretKey }}
```
Comment (alvarorm22) [inline chart/templates/workspace-api/secret.yaml:17]: `| quote`

**Review by alvarorm22 — CHANGES_REQUESTED** [review]
Add `| quote` in some secrets.

I think we should make the same changes in `secret.yaml` and `deployment.yaml` for the other components so that everyone load the secrets in the same way

**Comment by alberhander** [comment]
https://app.shortcut.com/cartoteam/story/213745/test-custom-gcs-buckets-with-k8

**Review by alvarorm22 — APPROVED** [review]
LGTM!

**Pull request merged**